README.md

```diff
@@ -158,6 +158,7 @@ If you use the server-side flow, Facebook will give you back a longer lived acce
 
 Actively tested with the following Ruby versions:
 
+- MRI 2.2.0
 - MRI 2.1.0
```

Comment:
Aren't we testing only the latest minor version? For example for 2.1 I can see there's 2.1.5 tested on CI.